src/cmd/scan.go

```diff
@@ -4,50 +4,532 @@
 package cmd
 
 import (
+	"context"
+	"encoding/json"
 	"fmt"
+	"io"
+	"log/slog"
+	"net/http"
 	"os"
+	"path/filepath"
+	"strings"
 
 	"github.com/defenseunicorns/uds-pk/src/compare"
+	"github.com/defenseunicorns/uds-pk/src/scan"
+	"github.com/defenseunicorns/uds-pk/src/utils"
+	"github.com/google/go-github/v73/github"
 	"github.com/spf13/cobra"
+	"github.com/zarf-dev/zarf/src/api/v1alpha1"
+	"go.yaml.in/yaml/v4"
+	"golang.org/x/oauth2"
 )
 
+var zarfYamlLocation string
```

Comment:
Similar comment about global flags vs structs holding all options for your command. See any [sample command in zarf](https://github.com/zarf-dev/zarf/blob/307d965b91bd261162da857a5caa2e64bac6c6ad/src/cmd/dev.go#L274-L323) for how we're doing it. In general global variables should be avoided as much as possible.